editor/filesystem_dock.cpp

```diff
@@ -2588,6 +2586,10 @@ void FileSystemDock::_change_split_mode() {
 	emit_signal(SNAME("display_mode_changed"));
 }
 
+void FileSystemDock::_split_dragged(int p_offset) {
+	split_box->is_vertical() ? split_box_offset_v = p_offset : split_box_offset_h = p_offset;
```

Comment:
```suggestion
	if (split_box->is_vertical()) {
		split_box_offset_v = p_offset;
	} else {
		split_box_offset_h = p_offset;
	}
```
Don't do assignment in ternary, it's unsafe and has poor readability